extensions/cornerstone/src/init.tsx

```diff
@@ -248,19 +248,19 @@ export default async function init({
 
     const activeTools = toolbarService.getActiveTools();
 
-    activeTools.forEach(tool => {
-      if (!toolGroup?._toolInstances?.[tool]) {
+    activeTools.forEach(toolId => {
+      if (!toolGroup?._toolInstances?.[toolId]) {
```

Comment:
Please access via some sort of getter - retrieving the tool directly makes it hard to change the representation in the future.